contracts/DKVDaggerHashimoto.sol

```diff
@@ -20,6 +20,9 @@ contract DecentralizedKVDaggerHashimoto is DecentralizedKV {
         uint256 coinbaseShare; // 10000 = 1.0
         ISystemContractDaggerHashimoto systemContract;
     }
+
+    bytes32 public constant EMPTY_CHUNK_4KB_HASH = 0xa8bae11751799de4dbe638406c5c9642c0e791f2a65e852a05ba4fdf0d88e3e6;
```

Comment:
I recommend to compute this parameter to be computed in constructor, once and only once, rather than hard coded here, since we might change the hash function. If the gas consumption is really a concern. At least we have to leave a comment here to show, which hash function has been adapted here.

Reply:
More useful advice for me.